presto-docs/src/main/sphinx/release/release-344.md

```diff
@@ -0,0 +1,64 @@
+# Release 344 (XX Oct 2020)
+
+## General Changes
+
+* Add {func}`murmur3` function. ({issue}`5054`)
+* Add {func}`from_unixtime_nanos` function. ({issue}`5046`)
+* Add [T-Digest](/functions/tdigest) type and functions. ({issue}`5158`)
+* Improve performance and latency of queries leveraging dynamic filters. ({issue}`5081`)
+* Allow to configure (using ``dynamic-filtering.service-thread-count`` feature toggle) number of
+  threads used for processing dynamic filters on coordinator. ({issue}`5341`)
+* Extend environment variables substitution (`${ENV:variable-name}`) to work also when the substituted text
+  is part of the configuration value. ({issue}`4345`)  
+* Remove `fast-inequality-joins` configuration property. The fast inequality joins remain enabled. ({issue}`5375`)
+* Report dynamic filters collection duration. ({issue}`5434`)  
+* Improve query performance by propagating more detailed dynamic filters to coordinator. ({issue}`5340`)  
```

Comment:
is this different from "Improve performance and latency of queries leveraging dynamic filters" above?
or, is one subset of the other?

(i guess these are separate improvements  in the code, but how do we present them as separate to the users?)

Reply:
DF improvements notes should most likely be grouped together.
Maybe we can replace both points as:
```
Improve query performance by propagating (with lower latency) more detailed dynamic filters to coordinator.
```
?

Reply:
How can i determine I am potentially affected?
For example, can https://prestosql.slack.com/archives/CGB0QHWSW/p1602211393325800 be related?

Reply:
This is also too much implementation detail. I would keep it high level and let people navigate to the issue if they want to understand what's happening under the covers. I'll fold the two items into one and link to both issues.